pallets/cash/src/lib.rs

```diff
@@ -411,6 +411,7 @@ fn has_requisite_signatures(notice_state: NoticeState, validators: &Vec<Validato
     match notice_state {
         NoticeState::Pending { signature_pairs } => match signature_pairs {
             ChainSignatureList::Eth(signature_pairs) => {
+                /// XXX inefficient
```

Comment:
Meant more like this https://github.com/compound-finance/gateway/blob/develop/pallets/cash/src/core.rs#L116 but I'll fix it